### ISSUE TITLE
fork error

### ISSUE THREAD

**Issue by realtimeradio** [ovro-lwa/lwa-issues#93]
Newly provisioned gpu servers 4,6,8 throw the following error when trying to start the digital pipeline 

```
Mar 03 14:36:22 lxdlwagpu08 lwa352_start_xeng_etcd_service.py[2444]: E0303 14:36:22.106158350   17133 fork_posix.cc:70]           Fork support is only compatible with the epoll1 and poll polling strategies
```

Looking at the installed `lwa352_start_xeng_etcd_service.py` script, the working machines contain:

![image](https://user-images.githubusercontent.com/40126647/156587195-c98da880-c000-4c1c-96ca-2d859fc929f6.png)

The non-working machines contain

![image](https://user-images.githubusercontent.com/40126647/156587289-61fe4079-b723-4934-b7de-38f697a0fd40.png)
[etc]

Reinstalling the control library with
```
cd ~/proj/lwa-shell/caltech-bifrost-dsp/pipeline-control
python setup.py install
```

results in the working configuration. Is the MAAS provisioning script installing via one of the many other python mechanisms?

**Comment by realtimeradio** [comment]
Scratch that! The error persists, even though the install now looks the same for all machines.

Googling suggests this was introduced in a newer version of `grpcio`. See, eg.  (https://github.com/ray-project/ray/issues/22518)

Working on a repo fix to force grpcio v1.43

**Comment by realtimeradio** [comment]
OK, well:
